configs/prometheus/config.yml

```
@@ -69,6 +69,14 @@ scrape_configs:
         app: robotoff
         service: api
         env: net
+  
+  - job_name: events-net
+    static_configs:
+    - targets: ['10.1.0.200:8000']
+      labels:
+        app: events
+        service: api
+        env: net
 
   - job_name: monitoring-org
     static_configs:
```

I think we need a line in `scrape_configs` / `static_configs`:
```suggestion
    static_configs:
    - targets: ['10.1.0.200:9171']
      labels:
        app: github_exporter
        service: metrics
        env: org
```